source/_layouts/documentation.blade.php

```diff
@@ -1,62 +1,71 @@
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/helper-methods/">Helper Methods</a>
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/page-metadata/">Page Metadata</a>
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/environments/">Environments</a>
-
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/collections/">Collections</a>
-                <a class="nav-list-item nav-list-item--sub" href="{{ $page->asset_prefix }}/docs/collections-extending-parent-templates/">Extending Parent Templates</a>
-                <a class="nav-list-item nav-list-item--sub" href="{{ $page->asset_prefix }}/docs/collections-paths/">Paths</a>
-                <a class="nav-list-item nav-list-item--sub" href="{{ $page->asset_prefix }}/docs/collections-sorting/">Sorting</a>
-                <a class="nav-list-item nav-list-item--sub" href="{{ $page->asset_prefix }}/docs/collections-pagination/">Pagination</a>
-                <a class="nav-list-item nav-list-item--sub" href="{{ $page->asset_prefix }}/docs/collections-variables-and-functions/">Variables &amp; Helper Functions</a>
-                <a class="nav-list-item nav-list-item--sub" href="{{ $page->asset_prefix }}/docs/collections-remote-collections/">Remote Collections</a>
-
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/pretty-urls/">Pretty URLs</a>
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/custom-404-page/">Custom 404 Page</a>
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/event-listeners/">Event Listeners</a>
-                <a class="nav-list-item" href="{{ $page->asset_prefix }}/docs/deploying-your-site/">Deploying Your Site</a>
-            </nav>
-            <p class="p-xs-x-1 p-xs-y-4 text-dark-soft">
-                <small>Brought to you by <a href="https://tighten.co" class="link-tighten">Tighten</a></small>
+
+        <div class="w-1/3 hidden items-center pl-3 lg:flex lg:pl-8">
+            <a href="https://github.com/tightenco/jigsaw" title="Contribute to Jigsaw on GitHub"
+                class="mr-6 text-blue-darker">
+                <img src="/assets/img/GitHub.svg" alt="GitHub alien logo">
+            </a>
+
+            <p class="text-sm text-blue-dark mb-0">
+                A project by
+                <a href="https://tighten.co" title="Tighten | Product Development for Web + Mobile"
+                    class="text-purple">Tighten</a>
             </p>
         </div>
-        <div class="col-md-9 documentation-page">
+
+        <navigation-toggle></navigation-toggle>
+    </nav>
+</header>
+
+
+<div class="bg-brown-lightest min-h-screen pt-32">
+    <div class="flex flex-col lg:flex-row py-2 container-content">
+        <navigation navigation-links="{{ $page->docsNavigation }}"></navigation>
```

Comment:
It would be better to pass this through as JSON rather than stringify it and then re-convert it inside the component.
`links='@json($page->docsNavigation)'`
should work...note the single quote.